packages/pharos-site/static/guidelines/checkbox.docs.mdx

```diff
@@ -75,23 +70,25 @@ import BestPractices from '../../src/components/statics/BestPractices.tsx';
         Labels are descriptive and succinct. They should provide further clarity for the user.
       </li>
       <li>Labels should not end in punctuation.</li>
+      <li>Use Sentence case for labels.</li>
       <li>
-        Use Sentence case for labels. Avoid using negative language as they can be counterintuitive.
-        For example, "I agree to the terms" instead of "I don't agree to the terms."
+        Avoid using negative language as they can be counterintuitive. For example, "I agree to the
```

Comment:
"They" is being used to refer to "negative language" which should be "it" instead:

```suggestion
        Avoid using negative language as it can be counterintuitive. For example, "I agree to the
```